### ISSUE TITLE
Me Controller Animation Very Slow

### ISSUE THREAD

**Issue by KangyKSLG** [AppliedEnergistics/Applied-Energistics-2#4181]
**Describe the bug**
After setting up and powering my controller it lit up and looked like it was showing it was invalid (red colour) but after looking for a bit the colours are changing but no where near as fast as default.

**To Reproduce**
Set up a controller and give it power

**Expected behavior**
ME Controller animation should cycle through colours fairly quickly

**Additional context**
Video of issue here: https://1drv.ms/v/s!AtsfINBFkWrdgtFio49b66VqKPe5KQ?e=9aKxt5

**Environment**
Refined Storage network in place as migrating from that (getting dismantled soon). Only connection ae2 has is an RS external storage to import all items to ae2. AE2 network is small, 20 64K drives, 20 ME interfaces and 20 molecular assemblers but no patterns (issue happens with just controller). Got 2 Mekanism ore quadrupling setups but they aren't running constantly. Various EnderIO, Thermal and Industrial foregoing machines for automation.  

- Minecraft Version: 1.12.2
- AE2 Version: rv6 stable 7
- Forge Version: 14.23.5.2847

**Comment by mindforger** [comment]
just as an reminder, since you did not mention them

make sure to install a proper amount of energy cells in your ME system, because input is limited by ME energy capacity/tick, regardless of how much you provide!

Maybe your system is showing a symptom of insufficient power and bootlooping in a pattern that let the animation appear delayed? Are you having any issues using the system?! delayed items transfer when moving whole stacks or crafting getting stuck?

**Comment by KangyKSLG** [comment]
I haven't noticed any issues, I moved around 100k of items from RS into my ME drives with no obvious issues. I have a big reactor powering my controller and 1 dense energy cell as a buffer. 

I'm not massively clued up on AE2 so unsure about how many energy cells are needed as a buffer but I can certainly add some more to see if that changes things

**Comment by mindforger** [comment]
nah sounds fine to me, i'd expect some subtile issues with item retrieval and autocrafting

i did not yet encounter this and i am runing the same versions but other mods around it

is this in any way reproducible in smaller scale?!

**Comment by KangyKSLG** [comment]
That's a good question, i'll load up world with just AE2 and see if I get the same results

**Comment by KangyKSLG** [comment]
Okay, loaded a test world with just AE2 enabled, placed down a ME controller and a creative energy cell with the same results.

**Comment by mindforger** [comment]
Hmmmm ... optifine or any other "optimizing" mods installed by chance? I could not reproduce this :( and unfotunately i can not test again, i am on mobile now :P

**Comment by KangyKSLG** [comment]
At the time I tested it was only AE2 installed. I've just created a new instance on MultiMC to test on a fresh install (same java version, same JVM flags, same forge version etc) and I couldn't reproduce it on that instance. Looks like there is something going on with my main instance. I'll close the issue as it seems to be something i've done or a knock on effect from.. Something.

Thanks for your time and sorry for wasting it. I'll close the issue.
